csm/conf/csm_cleanup.py

```diff
@@ -84,33 +84,40 @@ def process_stats(args):
     es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
     remove_old_indexes(es, args.d, args.n, args.e)
 
-def process_auditlogs(args):
+def process_es_cleanup(args):
```

Comment:
Is process_stats required now,  if not remove it.

Reply:
Comment addressed. Please review.